tests/integration_tests/data_steward/utils/participant_summary_requests_test.py

```diff
@@ -0,0 +1,189 @@
+        actual_access_token = psr.get_access_token()
+
+        # post conditions
+        self.assertEqual(actual_access_token, expected_access_token)
+
+    @mock.patch('utils.participant_summary_requests.requests.get')
+    def test_get_participant_data(self, mock_get):
+        """
+        Mocks calling the participant summary api.
+        """
+        # pre conditions
+        mock_get.return_value.status_code = 200
+        mock_get.return_value.json.return_value = self.json_response_entry
+
+        # test
+        expected_response = psr.get_participant_data(self.url, self.headers)
+
+        # post conditions
+        self.assertEqual(expected_response, self.participant_data)
+
+    def test_get_deactivated_participants_parameters(self):
+        """
+        Ensures error checking is working.
+        """
+        # Parameter check tests
+        self.assertRaises(RuntimeError, psr.get_deactivated_participants, None,
+                          self.dataset_id, self.tablename, self.columns)
+        self.assertRaises(RuntimeError, psr.get_deactivated_participants,
+                          self.project_id, None, self.tablename, self.columns)
+        self.assertRaises(RuntimeError, psr.get_deactivated_participants,
+                          self.project_id, self.dataset_id, None, self.columns)
+        self.assertRaises(RuntimeError, psr.get_deactivated_participants,
+                          self.project_id, self.dataset_id, self.tablename,
+                          None)
+
+    @mock.patch('utils.participant_summary_requests.requests.get')
+    def test_get_deactivated_participants(self, mock_get):
+        # Pre conditions
+        mock_get.return_value.status_code = 200
+        mock_get.return_value.json.return_value = self.json_response_entry
+
+        # Tests
+        psr.get_deactivated_participants(self.project_id, self.dataset_id,
+                                         self.tablename, self.columns)
+
+        # Post conditions
+        values = [(111, 'NO_CONTACT', parser.parse('2018-12-07T08:21:14 UTC')),
+                  (222, 'NO_CONTACT', parser.parse('2018-12-07T08:21:14 UTC'))]
+        self.assertTableValuesMatch(
+            '.'.join([self.project_id, self.destination_table]), self.columns,
+            values)
+
+    def test_store_participant_data(self):
+        # Parameter check test
+        self.assertRaises(RuntimeError, psr.store_participant_data,
+                          self.fake_dataframe, None, self.destination_table)
+
+        # Test
+        psr.store_participant_data(self.fake_dataframe, self.project_id,
+                                   self.destination_table)
+
+        # Post conditions
+        values = [(111, 'NO_CONTACT', parser.parse('2018-12-07T08:21:14 UTC')),
+                  (222, 'NO_CONTACT', parser.parse('2018-12-07T08:21:14 UTC'))]
+        self.assertTableValuesMatch(
+            '.'.join([self.project_id, self.destination_table]), self.columns,
+            values)
```

Comment:
If we go with `person_id` in the big query table, this will have to be changed also.
```suggestion
        bq_columns = ['person_id] + self.columns[1:]
        self.assertTableValuesMatch(
            '.'.join([self.project_id, self.destination_table]), bq_columns,
            values)
```

Reply:
Changed 👍🏾